ui/apps/platform/src/services/NetworkService.js

```diff
@@ -364,9 +363,7 @@ export function fetchNetworkPoliciesByClusterId(clusterId, deploymentQuery) {
         method: 'GET',
         url: `${networkPoliciesBaseUrl}?${params}`,
     };
-    return axios(options).then((response) => {
-        return sortBy(response?.data?.networkPolicies, 'name') ?? [];
-    });
+    return axios(options).then((response) => response.data.networkPolicies ?? []);
```

Comment:
Moves the responsibility of sorting to the calling function. Unfortunately this API does not support pagination or sorting via the backend.